src/Components/Footer.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import '../CSS/Footer.css'
+
+const Footer = () => {
```

Comment:
I like the idea of having a Footer as it's own component if there's going to be links to other resources or copyright information, etc.  I'm curious what you have in mind to render here?